mysql/datadog_checks/mysql/const.py

```diff
@@ -235,6 +235,9 @@
     'wsrep_flow_control_sent': ('mysql.galera.wsrep_flow_control_sent', MONOTONIC),
     'wsrep_cert_deps_distance': ('mysql.galera.wsrep_cert_deps_distance', GAUGE),
     'wsrep_local_send_queue_avg': ('mysql.galera.wsrep_local_send_queue_avg', GAUGE),
+    'wsrep_replicated_bytes': ('mysql.galera.wsrep_replicated_bytes', GAUGE),
+    'wsrep_received_bytes': ('mysql.galera.wsrep_received_bytes', GAUGE),
+    'wsrep_local_state_comment': ('mysql.galera.wsrep_local_state_comment', GAUGE),
```

Comment:
Example of values:
```
MariaDB [(none)]> show status like 'wsrep%';
+-------------------------------+-----------------------------------------------------+
| Variable_name                 | Value                                               |
+-------------------------------+-----------------------------------------------------+
[...]
| wsrep_local_state             | 4                                                   |
| wsrep_local_state_comment     | Synced                                              |
[...]
```

The comment doesn't look like a good value for a metric.
I think we should use the `wsrep_local_state` metric (which is always a integer) instead of the comment.